pkgs/by-name/gr/gradle-dependency-tree-diff/package.nix

```
@@ -0,0 +1,64 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  gradle,
+  makeBinaryWrapper,
+  jre_headless,
+  zulu11,
+  nix-update-script,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gradle-dependency-tree-diff";
+  version = "1.2.1";
+  src = fetchFromGitHub {
+    owner = "JakeWharton";
+    repo = "dependency-tree-diff";
+    tag = finalAttrs.version;
+    hash = "sha256-7ObmZygzSp7aAnqsJuMcPk+I3z993kjHCJMug3JkONg=";
+  };
+
+  nativeBuildInputs = [
+    gradle
+    makeBinaryWrapper
+  ];
+
+  mitmCache = gradle.fetchDeps {
+    inherit (finalAttrs) pname;
+    data = ./deps.json;
+  };
+  __darwinAllowLocalNetworking = true;
+
+  # There is a requirement on the specific Java toolchain.
+  gradleFlags = [ "-Dorg.gradle.java.home=${zulu11}" ];
+
+  gradleBuildTask = "build";
+  doCheck = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm644 build/dependency-tree-diff.jar \
+      $out/share/dependency-tree-diff/dependency-tree-diff.jar
+    makeWrapper ${lib.getExe jre_headless} $out/bin/dependency-tree-diff \
+      --add-flags "-jar $out/share/dependency-tree-diff/dependency-tree-diff.jar"
+
+    runHook postInstall
+  '';
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "An intelligent diff tool for the output of Gradle's dependencies task";
```

```suggestion
    description = "Intelligent diff tool for the output of Gradle's dependencies task";
```
https://nixos.org/manual/nixpkgs/unstable/#var-meta-description